Tests/SwiftDocCTests/Converter/RenderNodeCodableTests.swift

```diff
@@ -72,7 +72,23 @@ class RenderNodeCodableTests: XCTestCase {
             XCTFail(error.localizedDescription)
         }
     }
-    
+
+    func testSortedKeys() {
```

Comment:
If this test was declared as throwing then it wouldn't need its own error handling
```suggestion
    func testSortedKeys() throws {
```